library/pkparse.c

```diff
@@ -1297,7 +1297,9 @@ int mbedtls_pk_parse_key( mbedtls_pk_context *pk,
 
     mbedtls_pk_free( pk );
 
-    if( ret == MBEDTLS_ERR_PK_PASSWORD_MISMATCH )
+    if( ( ret == MBEDTLS_ERR_PK_PASSWORD_MISMATCH ) ||
+        ( ret == MBEDTLS_ERR_PKCS5_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_DES_FEATURE_UNAVAILABLE ) )
```

Comment:
When the key file is encrypted with DES, and the specific feature in DES is not supported by the acceleration

Reply:
In cases where it's not PKCS5.
For example:
~~~
PKCS#8 encrypted v2 PBKDF2 DES hmacWithSHA256
~~~

Reply:
Could also be AES or any other cipher, depending on the input data.

Reply:
I only added the errors that rose when I set different `FEATURE_UNAVAILABLE` in different modules

Reply:
As mentioned before, since we only support DES in our encrypted private key files ( and RC ), I don't the overhead of another error, which we don't test